fuse/eval/metrics/sequence_gen/metrics_seq_gen_common.py

```diff
@@ -18,13 +18,99 @@
 """
 from typing import Optional, Tuple, List
 from functools import partial
-
+from copy import copy
 import torch
 import numpy as np
 
 from fuse.eval.metrics.metrics_common import MetricPerBatchDefault
 
 
+class MetricCountSeqAndTokens(MetricPerBatchDefault):
```

Comment:
General question:
I'm not sure counting the sequences and tokens should be defined as metric. I don't have another suggestion it's just sounds weird :) 

What do you think of that?

Reply:
It uses the metric mechanism, and it's ok to me that it just counts some stats.